media-types.md

```diff
@@ -3,6 +3,7 @@
 The following `mediaType` MIME types are used by the formats described here, and the resources they reference:
 
 - `application/vnd.oci.descriptor.v1+json`: [Content Descriptor](descriptor.md)
+- `application/vnd.oci.descriptor.foreign.v1+json`: [Content Descriptor](descriptor.md)
```

Comment:
On Tue, Aug 30, 2016 at 02:39:23PM -0700, Stephen Day wrote:
> > @@ -3,6 +3,7 @@
> >  The following `mediaType` MIME types are used by the formats described here, and the resources they reference:
> >  
> >  - `application/vnd.oci.descriptor.v1+json`: [Content Descriptor](descriptor.md)
> > +- `application/vnd.oci.descriptor.foreign.v1+json`: [Content Descriptor](descriptor.md)
> 
> There is no `application/vnd.oci.descriptor.foreign.v1+json`. That doesn't exist.

It's in this PR, in the lines we're discussing now ;).  Maybe we
should table this sub-thread until @runcom rerolls the PR?